IdentifiabilityODE.mpl

```diff
@@ -557,3 +587,168 @@ CompareDiffVar := proc(dvl, dvr, var_list)
   end if:
   return StringTools[Compare](vr, vl):
 end proc:
+
+
+###
+# Weights
+###
+
+# get rid of unerscore, e.g. x1(t) -> x1_ -> x1
+get_function_name := f -> parse(convert(FunctionToVariable(f), string)[..-2]):
+
+# check if depends on (t)
+is_function:= f->StringTools[Has](convert(f, string), "(t)"):
+idtfm := x->x;
+
+# check if is derivative
+is_diff := f->type(int(f, t), function(name)):
+
+lhs_name := ff -> if convert(ff, string)[-1] = "_" then parse(convert(ff, string)[..-2]) else ff; end if:
+
+get_state_name := proc(state, x_vars, mu)

```

Comment:
I suggest to use the same style of highlighting the function declaration as used before